src/main/java/us/icarenow/web/model/entity/Patient.java

```diff
@@ -0,0 +1,82 @@
+package us.icarenow.web.model.entity;
+
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="icn_patient")
+public class Patient {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    private int userId;
+
+    private String cnp;
+
+    private String firstName;
+
+    private String lastName;
+
+    private boolean insured;
```

Comment:
@andreimladin eu in sql nu am boolean si am pus bit, oare daca e asa e bai?

Reply:
hai sa il lasam boolean